### PR TITLE
add missing entries to ListProjectOptions

### LINKED ISSUE #857: Add missing query parameters for Projects
https://docs.gitlab.com/ce/api/projects.html#list-projects lists numerous query options for the api that are not available in `ListProjectsOptions`, such as `last_activity_after` and `last_activity_before` among others.

### PR DESCRIPTION
**Pull request by razor-1**
Resolves #857

### CODE REVIEW

File: projects.go [hunk 1]
```diff
@@ -228,6 +228,12 @@ type ListProjectsOptions struct {
 	MinAccessLevel           *AccessLevelValue `url:"min_access_level,omitempty" json:"min_access_level,omitempty"`
 	WithCustomAttributes     *bool             `url:"with_custom_attributes,omitempty" json:"with_custom_attributes,omitempty"`
 	WithProgrammingLanguage  *string           `url:"with_programming_language,omitempty" json:"with_programming_language,omitempty"`
+	WikiChecksumFailed       *bool             `url:"wiki_checksum_failed,omitempty" json:"wiki_checksum_failed,omitempty"`
+	RepositoryChecksumFailed *bool             `url:"repository_checksum_failed,omitempty" json:"repository_checksum_failed,omitempty"`
+	IdAfter                  *int              `url:"id_after,omitempty" json:"id_after,omitempty"`
+	IdBefore                 *int              `url:"id_before,omitempty" json:"id_before,omitempty"`
+	LastActivityAfter        *ISOTime          `url:"last_activity_after,omitempty" json:"last_activity_after,omitempty"`
+	LastActivityBefore       *ISOTime          `url:"last_activity_before,omitempty" json:"last_activity_before,omitempty"`
```
Comment (svanharmelen) [inline projects.go:236]: These need to be `*time` instead of `*ISOTime` if I read the docs correctly.

File: projects.go [hunk 2]
```diff
@@ -228,6 +228,12 @@ type ListProjectsOptions struct {
 	MinAccessLevel           *AccessLevelValue `url:"min_access_level,omitempty" json:"min_access_level,omitempty"`
 	WithCustomAttributes     *bool             `url:"with_custom_attributes,omitempty" json:"with_custom_attributes,omitempty"`
 	WithProgrammingLanguage  *string           `url:"with_programming_language,omitempty" json:"with_programming_language,omitempty"`
+	WikiChecksumFailed       *bool             `url:"wiki_checksum_failed,omitempty" json:"wiki_checksum_failed,omitempty"`
+	RepositoryChecksumFailed *bool             `url:"repository_checksum_failed,omitempty" json:"repository_checksum_failed,omitempty"`
+	IdAfter                  *int              `url:"id_after,omitempty" json:"id_after,omitempty"`
+	IdBefore                 *int              `url:"id_before,omitempty" json:"id_before,omitempty"`
```
Comment (svanharmelen) [inline projects.go:234]: In Go these would be written as `IDAfter` and `IDBefore`: https://github.com/golang/go/wiki/CodeReviewComments#initialisms

File: projects.go [hunk 3]
```diff
@@ -228,6 +228,12 @@ type ListProjectsOptions struct {
 	MinAccessLevel           *AccessLevelValue `url:"min_access_level,omitempty" json:"min_access_level,omitempty"`
 	WithCustomAttributes     *bool             `url:"with_custom_attributes,omitempty" json:"with_custom_attributes,omitempty"`
 	WithProgrammingLanguage  *string           `url:"with_programming_language,omitempty" json:"with_programming_language,omitempty"`
+	WikiChecksumFailed       *bool             `url:"wiki_checksum_failed,omitempty" json:"wiki_checksum_failed,omitempty"`
+	RepositoryChecksumFailed *bool             `url:"repository_checksum_failed,omitempty" json:"repository_checksum_failed,omitempty"`
```
Comment (svanharmelen) [inline projects.go:232]: I know you didn't add it there, but could you move `MinAccessLevel` below `RepositoryChecksumFailed` to keep the fields inline with the ordering in the docs? Thanks!

File: projects.go (hunk 2 above)
Comment (razor-1) [reply projects.go]: duh, my bad. thanks

File: projects.go (hunk 1 above)
Comment (razor-1) [reply projects.go]: you're right; corrected

File: projects.go (hunk 3 above)
Comment (razor-1) [reply projects.go:231]: updated all ordering to match the docs

**Pull request merged**